docs/src/pages/customization/palette/palette.md

```diff
@@ -177,3 +177,7 @@ const theme = createMuiTheme({
 ```
 
 {{"demo": "pages/customization/palette/DarkTheme.js"}}
+
+## The default values
+
+You can explore all the default values of the palette using our [explorer](/customization/default-theme/?expend-path=$.palette) or by opening the dev tools console on this page (`window.theme.palette`).
```

Comment:
Good timing – only in the last few days while looking at the "default" palette, I noticed that it is not actually showing the default values, but rather those used in the docs (including the current selection for  colors,  light / dark & LTR.

Do we fix it so it shows the default, or change the title and wording of that page and these comments?

Reply:
https://github.com/mui-org/material-ui/blob/fdc7de2d5ec95873a48f37937c1b3bb35d864773/docs/src/pages/customization/default-theme/DefaultTheme.js#L48-L53